docs/how-to-guides/working-with-images.md

```diff
@@ -23,11 +23,12 @@ You can learn about all of the command options and display the help documentatio
 nerdctl -h
 ```
 
-The initial set of images are stored in the same containerd that Kubernetes uses,
-and are part of the `k8s.io` namespace. You can also switch to a namespace called
-`default` if you wish to build or pull images into a different namespace. Currently
-the only way to create other namespaces is to build or pull an image with the
-`nerdctl` CLI, using the `--namespace <NAMESPACE_NAME>` option.
+Unlike docker, containerd features its own namespaces, by default images are stored in the `default` namespace.
```

Comment:
Can we add these statements without removing the earlier statements, as I feel your steps/statements compliment the original information here.